framework/core/src/Notification/MailableInterface.php

```diff
@@ -15,11 +15,26 @@ interface MailableInterface
 {
     /**
      * Get the name of the view to construct a notification email with.
+     *
+     * To provide the best experince for the user, provide both a `text` and `html` view.
+     *
+     * Example:
+     * ```php
+     * return [
+     *  'text' => 'flarum-subscriptions::emails.plain.newPost',
+     *  'html' => 'flarum-subscriptions::emails.html.newPost'
+     * ];
+     * ```
      */
-    public function getEmailView(): string|array;
+    public function getEmailView(): array;
```

Comment:
Can we use PHPStan's docblock syntax to annotate the returned shape? https://phpstan.org/writing-php-code/phpdoc-types#array-shapes

Reply:
Thanks for the tip, I wasn't aware of this :)